src/api/types/generated/StopsDetailsQuery.ts

```diff
@@ -14,6 +14,15 @@ export type StopsDetailsQuery = {
       name: string;
       publicCode?: string;
       stopPlace?: {id: string};
+      situations: Array<{
+        situationNumber?: string;
+        reportType?: Types.ReportType;
+        summary: Array<{language?: string; value: string}>;
+        description: Array<{language?: string; value: string}>;
+        advice: Array<{language?: string; value: string}>;
+        validityPeriod?: {startTime?: any; endTime?: any};
+        infoLinks?: Array<{uri: string; label?: string}>;
+      }>;
```

Comment:
Should this be added now? This doesn't reflect what's in the BFF.

Reply:
Removed